crates/typst-library/src/foundations/mod.rs

```diff
@@ -66,6 +66,7 @@ pub use self::target_::*;
 pub use self::ty::*;
 pub use self::value::*;
 pub use self::version::*;
+use comemo::Track;
 pub use typst_macros::{scope, ty};
 use typst_syntax::SyntaxMode;
```

Comment:
could you move the import don't below to the existing `use comemo::TrackedMut`? might as well fix the typst_syntax one while at it.